server/routes/gitlab/gitlab_routes.py

```
@@ -0,0 +1,342 @@
+    """Check if an org-level GitLab token is configured."""
+    try:
+        creds = get_credentials_from_db(user_id, "gitlab")
+        if creds and creds.get("access_token"):
+            return jsonify({
+                "connected": True,
+                "username": creds.get("username", ""),
+                "base_url": creds.get("base_url", DEFAULT_GITLAB_URL),
+                "token_name": creds.get("token_name", ""),
+            })
+        return jsonify({"connected": False})
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)